index.js

```diff
@@ -0,0 +1,11 @@
+fetch('./deck.json')
+    .then(data => data.json())
+    .then((fetchDeck)=>{
+        let deck = fetchDeck.hippos
+        deck = randomiseDeck(deck)
+        assignCards(deck)
+        addClickEvents(deck)
+    }).catch(()=>{
+        alert('Something went wrong. Please refresh the page.')
```

Comment:
please change the message to 'Something went wrong. Please contact an administrator.'